actiontext/lib/action_text/plain_text_conversion.rb

```diff
@@ -18,9 +18,11 @@ def plain_text_for_node(node, index = 0)
       end
 
       def plain_text_for_node_children(node)
-        node.children.each_with_index.map do |child, index|
-          plain_text_for_node(child, index)
-        end.compact.join("")
+        texts = []
+        node.children.each_with_index do |child, index|
+          texts << plain_text_for_node(child, index)
+        end
+        texts.join("")
```

Comment:
Is not `compact` necessary? (`plain_text_for_node` will not return nil?)

Reply:
It doesn't appear to be necessary, and I don't see `nil` being returned anywhere. I don't think it's an issue either way since `nil`s would disappear as empty strings when the array is `join`ed:
```ruby
>> [ "a", nil, nil, "b", "", nil, "c" ].join("")
=> "abc"
```